src/codeyCommand.ts

```diff
@@ -292,11 +308,12 @@ export class CodeyCommand extends SapphireCommand {
           case CodeyCommandResponseType.STRING:
             return interaction.editReply(<string>successResponse);
         }
-      } catch (e) {
-        console.log(e);
-        return interaction.editReply(commandDetails.messageIfFailure!);
       }
+    } catch (e) {
+      console.log(e);
+      return await interaction.editReply(
+        commandDetails.messageIfFailure ?? 'Codey backend error - contact a mod for assistance.'
```

Comment:
I think it would be a good idea to put `'Codey backend error - contact a mod for assistance.'` into a constant since it is used more than once.

Reply:
This is now done